accname/name/comp_name_from_content.tentative.html

```diff
@@ -37,6 +37,27 @@ <h3 data-expectedlabel="image link" data-testname="heading with link referencing
   <img id="nested_image_label4" alt="image" aria-labelledby="nested_image_label4 crossref_link2" src="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==">
 </h3>
 
+
+<!-- “びょういんのかんじゃサービス” means “Hospital patient services” -->
+<!-- text-transform:full-size-kana visually changes: -->
+<!--   1. びょういん (byōin = hospital) into びよういん (biyōin = beauty parlor) -->
+<!--   2. かんじゃ (kanja = patient) into かんじや (kanjiya = [no meaning]) -->
+<!-- ...causing the whole heading to become nonsensical. -->
+<h1>heading which visually transforms <span lang="ja">びょういん</span> (hospital) to <span lang="ja">びよういん</span> (beauty parlor) using CSS <code>text-transform:full-size-kana</code></h1>
+<h3 lang="ja" data-expectedlabel="びょういんのかんじゃサービス" data-testname="heading name from content with text-transform:full-size-kana" class="ex" style="text-transform:full-size-kana;">びょういんのかんじゃサービス</h3>
+
+<!-- “竜の背に乗る” means “Ride the dragon’s back” -->
+<!-- text-transform:full-size-kana visually changes the <rt> furigana for 竜 -->
+<!-- from りゅう (ryū = dragon) to りゆう (riyuu = reason) -->
+<!-- TODO: Restore this subtest when ruby’s accname computation and -->
+<!-- text-to-speech requirements are settled. -->
+<!--
+<h1>link containing a <code>ruby > rt</code> element which visually transforms <span lang="ja">りゅう</span> (dragon) to <span lang="ja">りゆう</span> (reason) using CSS <code>text-transform:full-size-kana</code></h1>
+<button lang="ja" data-expectedlabel="竜の背に乗る" data-testname="ruby > rt name from content with text-transform:full-size-kana" class="ex" >
+  <ruby>竜<rp>(</rp><rt style="text-transform:full-size-kana;">りゅう</rt><rp>)</rp>の背<rp>(</rp><rt>せ</rt><rp>)</rp>に乗<rp>(</rp><rt>の</rt><rp>)</rp>る</ruby>
+</button>
+-->
```

Comment:
Ruby will require more than a single subtest, so let's keep this test but reformat it to avoid the current problems with testing AccName from Ruby. Something like this?

```suggestion
<button lang="ja" data-expectedlabel="りゅう" data-testname="button with text-transform: full-size-kana on contents" class="ex" style="text-transform: full-size-kana;">りゅう</button>
```